### PR TITLE
Docs-1830 / 21.04 /  Adding API test for scale container images

### PR DESCRIPTION
**Pull request by ericbsd**
added container API test to disable enable_image_updates
added test to pull container images
added test to verify enable_image_updates is false
added test to get a container image id
added test to get a container with the image id
added test to delete a container with the image id
added dependency to setup_kubernetes and added depends to pull_image
added pool_4 depends on test 02 of test_273_kubernetes.py
added dependency of setup_kubernetes to test 03, 04, 05

### CODE REVIEW

**Review by q5sys — APPROVED** [review]
LGTM

File: tests/api2/test_305_container.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    job_status = wait_on_job(job_id, 180)
+    assert job_status['state'] == 'SUCCESS', str(job_status['results'])
+
+
+def test_06_get_new_image_id(request):
+    depends(request, ["pull_image"])
+    global image_id
+    results = GET("/container/image/")
+    for result in results.json():
+        print(result)
```
Comment (sonicaj) [inline tests/api2/test_305_container.py:88]: Leftover debug ?

File: tests/api2/test_305_container.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    assert isinstance(results.json(), list), results.text
+
+
+@skip_container_image
+@pytest.mark.dependency(name='pull_image')
+def test_05_pull_container_image(request):
+    depends(request, ["setup_kubernetes"], scope="session")
+    global image_id
+    payload = {
+        "docker_authentication": {
```
Comment (sonicaj) [inline tests/api2/test_305_container.py:69]: How about testing publicly available images as well ?

File: tests/api2/test_305_container.py (hunk 1 above)
Comment (ericbsd) [reply tests/api2/test_305_container.py:88]: Yes thanks

File: tests/api2/test_305_container.py (hunk 2 above)
Comment (ericbsd) [reply tests/api2/test_305_container.py:69]: Got merged but yes I will make a new PR and add that.

**Pull request merged**